internal/executor/instance/persistentworker/isolation/vetu/vm.go

```diff
@@ -131,6 +136,11 @@ func (vm *VM) Start(
 			args = append(args, "--net-host")
 		}
 
+		// Apply "run" resource modifier
+		if vm.resourceModifier != nil {
```

Comment:
Maybe just extra caution?

```suggestion
		if vm.resourceModifier != nil && vm.resourceModifier.Append != nil {
```

Reply:
This won't compile because the `Append` can't be `nil` 🤔